src/assets/icons/notifications-off.svg

```diff
@@ -1,5 +1,5 @@
-<!-- https://material.io/icons/#ic_notifications_off -->
```

Comment:
I removed this reference link because it no longer seems to be relevant. 

It appears as if the Material Design icon sets have been modified since this link was added. 
see https://material.io/icons/#ic_notifications_off

![Screen Shot 2020-05-14 at 1 28 11 PM](https://user-images.githubusercontent.com/1427548/81982580-ee39bf00-95e6-11ea-8ebf-7d30a795ca9f.png)

![Screen Shot 2020-05-14 at 1 27 59 PM](https://user-images.githubusercontent.com/1427548/81982588-f134af80-95e6-11ea-86d0-f0e4ca250e24.png)